### DISCUSSION TITLE
How to do viewport crop?

### DISCUSSION THREAD

**Discussion by LuukVerhagen** [dlemstra/Magick.NET#625] [General] [answered]
As described in http://www.imagemagick.org/Usage/anim_mods/#crop_viewport, you should not only crop the individual frames, but also the whole viewport. In the command line version, this is possible by adding the '!' flag. I was wondering what the Magick.NET equivalent of this flag is?

**Comment by dlemstra** [comment, marked as answer]
You could do this: `var geometry = new MagickGeometry("50x50+3+30!")` or set the `IgnoreAspectRatio` property of the `geometry` to `true`.

**Reply by LuukVerhagen** [reply]
Hartelijk dank
